apps/simple-trading-app/src/app/components/deal-ticket/deal-ticket-container.tsx

```diff
@@ -1,13 +1,14 @@
-import type { DealTicketContainerProps } from '@vegaprotocol/deal-ticket';
 import {
   DealTicketManager,
   DealTicketContainer as Container,
 } from '@vegaprotocol/deal-ticket';
 import { DealTicketSteps } from './deal-ticket-steps';
+import { useParams } from 'react-router-dom';
 
-export const DealTicketContainer = ({ marketId }: DealTicketContainerProps) => {
+export const DealTicketContainer = () => {
+  const { marketId } = useParams();
```

Comment:
suggestion (non blocking):
```suggestion
  const { marketId } = useParams<{ marketId: string }>();
```
then no need for typecasting below